docs/en/plan-deployment/cost.md

```diff
@@ -42,20 +42,35 @@ Key assumptions include:
 - 10/100/1000RPS
 
-| Request Per Second | ALB | EC2 | Buffer type       | Buffer cost | S3   |  Total (USD/Month) |
-| ------------------ | --- | --- | ----------------- | ----------- | ---  |  --------- |
-| 10RPS             |   7  | 122    | Kinesis (On-Demand) |    36         |   3   |     168  |
-|                         |   7  |  122   | Kinesis (Provisioned 2 shard)   |      22       |  3   |   154  |
-|                         |   7  |  122   | MSK (m5.large * 2, connector MCU * 1)   |       417      |   3  |     549   |
-|                         | 7    |  122   | None              |             |  3    |      132   |
-|100RPS           |   43  |  122  |     Kinesis(On-demand)              |      86       |  3   |     254 |
-|                         | 43    |   122  | Kinesis (Provisioned 2 shard)   |      26       | 3    |     194  |
-|           |   43  |  122  |     MSK (m5.large * 2, connector MCU * 1)              |      417       |  3   |     585
-|           |   43  |  122  |     None              |             |  3    |     168
-|1000RPS           |   396  |   122 |     Kinesis(On-demand)              |      576       |  14   |    1108 |
-|                         |  396   |  122   | Kinesis (Provisioned 10 shard)   |    146         |   14  |     678  |
-|           |  396   | 122   |     MSK (m5.large * 2, connector MCU * 2~3)              |      530       |  14  |     1062
-|           |  396   | 122   |     None              |            |  14   |     532
+| Request Per Second | ALB | EC2  |  Buffer type      | Buffer cost | S3   |  Total (USD/Month) |
+| ------------------ | --- | ---  |  --------------   | ----------- | ---  |  --------- |
+| 10RPS              |  7  |  122 |  Kinesis (On-Demand) |    36       |   3  |     168  |
+|                    |  7  |  122 |  Kinesis (Provisioned 2 shard)   |      22       |  3   |   154  |
+|                    |  7  |  122 |  MSK (m5.large * 2, connector MCU * 1)   |       417      |   3  |     549   |
+|                         | 7    |  122 |  None              |             |  3    |      132   |
+|100RPS           |   43  |  122  |  Kinesis(On-demand)              |      86       |  3   |     254 |
+|                         | 43    |   122 |  Kinesis (Provisioned 2 shard)   |      26       | 3    |     194  |
+|           |   43  |  122  |   MSK (m5.large * 2, connector MCU * 1)              |      417       |  3   |     585
+|           |   43  |  122 |      None              |             |  3    |     168
+|1000RPS           |   396  |   122 |      Kinesis(On-demand)              |      576       |  14   |    1108 |
+|                         |  396   |  122  |  Kinesis (Provisioned 10 shard)   |    146         |   14  |     678  |
+|           |  396   | 122  |      MSK (m5.large * 2, connector MCU * 2~3)              |      530       |  14  |     1062
+|           |  396   | 122   |      None              |            |  14   |     532
+
+## Data transfer
```

Comment:
Some rephrase suggestions,

```markdown
## Data transfer
There are associated costs when data is transferred from EC2 to the downstream data sink. Below is an example of data transfer costs based on 1000 RPS and a 1KB request payload.

1. EC2 Network In: This does not incur any costs.
2. EC2 Network Out: There are three data sink options:
       - S3: It is recommended to use VPC endpoint, which does not incur any costs.
       - MSK: It will cost `$0.010 per GB in/out/between EC2 AZs`, about $105/month.
       - KDS: There are two ways to sink data from EC2 to KDS: NAT or VPC endpoint.
       - NAT: If there is one NAT, the total cost is about $633/month. The price consists of the following items:
               1. `$0.045 per NAT Gateway Hour`, $32/month.
               2. `$0.045 per GB Data Processed by NAT Gateways`, $601/month.
VPC Endpoint: $147/month.

We suggest using a [VPC endpoint](https://docs.aws.amazon.com/whitepapers/latest/aws-privatelink/what-are-vpc-endpoints.html) for the KDS data sink. For more information on using the VPC endpoint, please refer to the VPC endpoint documentation.
```

Reply:
Refactored it.